### ISSUE TITLE
Job Containment

### ISSUE THREAD

**Issue by grondo** [flux-framework/flux-core#4433]
This issue tracks support for containing jobs to the resources they've been allocated using cgroups or other mechanisms as required for using Flux as the native resource manager on shared nodes systems.

 - [] #4255
 - [] #3346 
 - [] flux-pam module or other remote login access control should restrict user to allocated resources on node

**Comment by trws** [comment]
@grondo @garlick is this still active as written? The pam module is still relevant I would think, but given the systemd containment it seems like this maybe needs a reword or a split.

**Comment by grondo** [comment]
Yeah, I think the flux-pam item is the only relevant one from above. {erhaps we should close and reopen a relevant one that details actual TODO items here.
